pkg/controller/node/node_controller_test.go

```diff
@@ -1046,44 +1053,49 @@ func TestShouldMakeProgress(t *testing.T) {
 			expectTaintsAddPatch:  false,
 		},
 		{
-			description:           "node not at desired config, patch on annotation and taints",
+			description:           "node not at desired config, patch on annotation and taints", // failing
```

Comment:
question: Is this test still failing? We may want to remove the `// failing` comment if so.